--- schroedinger-1.0.8.orig/configure.ac	2009-09-29 07:23:37.000000000 +0200
+++ schroedinger-1.0.8/configure.ac	2009-04-23 05:20:57.000000000 +0200
@@ -150,6 +150,47 @@
 SCHRO_CFLAGS="$SCHRO_CFLAGS -I\$(top_srcdir)"
 AC_SUBST(SCHRO_CFLAGS)
 
+########################
+# GStreamer stuff
+########################
+
+GST_MAJORMINOR=0.10
+AC_SUBST(GST_MAJORMINOR)
+dnl *** check for arguments to configure ***
+
+if test "x$with_opengl" = xyes; then
+  GST_GL_DEP="gst-plugins-gl-0.10"
+fi
+
+AC_ARG_ENABLE(gstreamer, AC_HELP_STRING([--disable-gstreamer],
+              [disable gstreamer support (default=enabled)]))
+if test "x$enable_gstreamer" != xno
+then
+  PKG_CHECK_MODULES(GST, gstreamer-0.10 >= 0.10.19 gstreamer-base-0.10 >= 0.10.19 gstreamer-video-0.10 >= 0.10.19 ${GST_GL_DEP},
+      HAVE_GST=yes, HAVE_GST=no)
+else
+  HAVE_GST=no
+fi
+
+AM_CONDITIONAL(HAVE_GST, test "x$HAVE_GST" = "xyes")
+AC_SUBST(GST_CFLAGS)
+AC_SUBST(GST_LIBS)
+
+plugindir="\$(libdir)/gstreamer-$GST_MAJORMINOR"
+AC_SUBST(plugindir)
+
+GST_ALL_LDFLAGS="-no-undefined"
+AC_SUBST(GST_ALL_LDFLAGS)
+
+GST_PLUGIN_LDFLAGS="-module -avoid-version -export-symbols-regex '^[_]*gst_plugin_desc\$\$' $GST_ALL_LDFLAGS"
+AC_SUBST(GST_PLUGIN_LDFLAGS)
+
+AC_DEFINE(GST_PACKAGE_ORIGIN, "unknown", [Package origin in GStreamer plugin])
+
+AC_ARG_ENABLE(video-compat, AC_HELP_STRING([--disable-video-compat],
+    [disable gstvideo compatibility (default=enabled)]))
+AM_CONDITIONAL(ENABLE_VIDEO_COMPAT, test "x$enable_video_compat" != "xno")
+
 
 ########################
 # Check for CUDA
@@ -238,6 +279,8 @@
 schroedinger/Makefile
 schroedinger/cuda/Makefile
 schroedinger/opengl/Makefile
+gst-libs/gst/video/Makefile
+gst/Makefile
 testsuite/Makefile
 testsuite/cache/Makefile
 testsuite/cuda/Makefile
--- schroedinger-1.0.8.orig/Makefile.am	2009-09-29 07:08:51.000000000 +0200
+++ schroedinger-1.0.8/Makefile.am	2009-04-22 04:35:57.000000000 +0200
@@ -4,7 +4,11 @@
 
 DISTCHECK_CONFIGURE_FLAGS=--enable-gtk-doc
 
-DIST_SUBDIRS = schroedinger doc tools testsuite
+if HAVE_GST
+SUBDIRS += gst-libs/gst/video gst
+endif
+
+DIST_SUBDIRS = schroedinger doc tools gst testsuite gst-libs/gst/video
 
 EXTRA_DIST = COPYING COPYING.GPL COPYING.LGPL COPYING.MIT COPYING.MPL \
 	     autogen.sh schroedinger.spec TODO NEWS
