diff -u schroedinger-1.0.9/Makefile.in schroedinger-1.0.9.b/Makefile.in
--- schroedinger-1.0.9/Makefile.in	2010-03-04 10:04:43.000000000 +0100
+++ schroedinger-1.0.9.b/Makefile.in	2010-03-05 20:47:06.000000000 +0100
@@ -290,9 +290,9 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 AUTOMAKE_OPTIONS = foreign
-SUBDIRS = schroedinger doc tools testsuite
+SUBDIRS = schroedinger doc tools
 DISTCHECK_CONFIGURE_FLAGS = --enable-gtk-doc
-DIST_SUBDIRS = schroedinger doc tools testsuite
+DIST_SUBDIRS = schroedinger doc tools
 EXTRA_DIST = COPYING COPYING.GPL COPYING.LGPL COPYING.MIT COPYING.MPL \
 	     autogen.sh schroedinger.spec TODO NEWS
 
